__init__.py

```diff
@@ -704,7 +690,8 @@ def validate_duration(string):
         wait_while_speaking()
         self.enable_intent("handle_mute_timer")
         # Start showing the remaining time on the faceplate
-        self.update_display(None)
+        if self.platform != "mycroft_mark_2":
```

Comment:
can we remove these platform checks? i can have a remote GUI connected to a mark1 or any other enclosure, just send the gui data and if theres is no GUI it will be ignored

Reply:
I'll move this to use self.gui.connected(), the gui check is needed as for mark-1 it would still use the old pathway to do a timer and for gui qml handles the timer

Reply:
i dont think even the connected is needed? in the case of a mark1 it should use both functions calls

- regular faceplate stuff
- gui messages for any remote GUI

the advantage of not using self.gui.connected is that if i connect with the timers already set they will still show up

Reply:
The render paths of timers is very different, qml manages and renders the timers for gui, no update data is sent from skill to qml, we only connect the timers signals to slots in the skill for events like expired timer etc, for qml, not adding a check here would have two unsynced timers running and causing all sorts of issues, so its better to disable one path and use one path exclusively if there is a gui.

Reply:
it's totally fine if the mark1 doesnt use the GUI then, but please check for platform==mark1  instead of platform != mark2